lib/mercadopago/resources/order.rb

```diff
@@ -0,0 +1,19 @@
+# typed: true
+# frozen_string_literal: true
+
+module Mercadopago
+  ###
+  # This class provides the methods to access the API that will allow you to create an order on your website
+
+  # From basic to advanced configurations, you control the whole experience.
+
+  # [Click here for more infos](https://www.mercadopago.com.br/developers/en/guides/online-payments/checkout-api/introduction/)
```

Comment:
[req] Atualizar URL e remover `.br`
```suggestion
  # [Click here for more infos](https://www.mercadopago.com/developers/en/docs/checkout-api/landing)
```